tuxemon/core/platform/platform_pygame/events.py

```diff
@@ -0,0 +1,217 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from collections import defaultdict
+
+import pygame as pg
+
+from tuxemon.core import prepare
+from tuxemon.core.platform.const import buttons, intentions, events
+from tuxemon.core.platform.events import InputHandler, PlayerInput, EventQueueHandler
+
+
+class PygameEventQueueHandler(EventQueueHandler):
+    """ Handle all events from the pygame event queue
+    """
+
+    def __init__(self):
+        self._inputs = defaultdict(list)  # type Dict[int, InputHandler]
```

Comment:
type should be `Dict[int, List[InputHandler]]` ?

Reply:
yeah, good catch